bookmarks/middlewares.py

```diff
@@ -1,6 +1,30 @@
 from django.conf import settings
 from django.contrib.auth.middleware import RemoteUserMiddleware
-
+from django.http import JsonResponse
+from django.db import connections
+from bookmarks.views.settings import app_version
 
 class CustomRemoteUserMiddleware(RemoteUserMiddleware):
     header = settings.LD_AUTH_PROXY_USERNAME_HEADER
+
+class HealthCheckMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if request.path == '/health':
```

Comment:
It looks like this isn't called when the database is unable to connect. In `siteroot/settings/base.py` I modified the SQLite path to something invalid, then started the app. Accessing `localhost:8000/health` returns `Not found`. It works if the connection params are correct.

Reply:
Seems that is only the case for the Django dev server, it works fine when running the app through uwsgi. Feel free to to disregard then.

Reply:
In my experience the entire app doesn't seem to start unless the DB connection is valid. Are you seeing the same?

Reply:
Yeah, something like that. Should be fine to ignore in the dev environment.